src/routes/(app)/elections/+page.svelte

```diff
@@ -29,8 +29,9 @@
         {#if isAuthorized(apiNames.ELECTION.UPDATE, data.user)}
           <a
             href={"/elections/" + election.id + "/edit"}
-            class="btn btn-secondary btn-sm ml-auto">Redigera</a
-          >
+            class="btn btn-outline btn-sm ml-auto"
```

Comment:
We have a lot of purple (`btn-secondary`) buttons on the page, but I'm not a big fan of them so I changed this one to a more neutral color.

I'm not really an expert on UI design (clearly... 😄) but, colors tend to draw attention and usually signify some kind of main action we want the user to take. Editing is more of an optional action and thus doesn't need to be highlighted.